pipeline-scripts/release.groovy

```diff
@@ -116,6 +116,22 @@ Map stageValidation(String quay_url, String dest_release_tag, int advisory = 0,
         error("🚫 Advisory ${advisoryInfo.id} doesn't seem to be associated with a live ID.")
     }
 
+    if (arch == 'amd64' || arch == 'x86_64') {
+        echo "Verifying payload"
+        res = commonlib.shell(
+            returnAll: true,
+            script: "elliott --group=openshift-${version} verify-payload registry.svc.ci.openshift.org/ocp/release:${nightly} ${advisory}"
+        )
+        if (res.returnStatus != 0) {
+            def cd = currentBuild.description
+            currentBuild.description = "${currentBuild.description} - INPUT REQUIRED"
+            slackChannel = slacklib.to(version)
+            slackChannel.failure("Verify-payload failed. User input required to proceed")
+            input 'Advisory contents does not match payload. Proceed anyway?'
```

Comment:
I'm not sure there's a place for the user to check what didn't match - does that show up in the console logs? Anyway it would help to include the output from `verify-payload` (it creates a json file) in this user prompt.